src/rt/sections_osx.d

```diff
@@ -23,6 +23,13 @@ import core.sys.osx.mach.getsect;
 import rt.deh, rt.minfo;
 import rt.util.container.array;
 
+version (D_LP64)
+    import rt.sections_osx64;
+else
+    import rt.sections_osx32;
+
+mixin SectionsOSX;
```

Comment:
Actually, taking a fresh look at things this morning, these three files are so short that I don't think it really makes sense to split them up. Can we just keep it all in one file?